src/python/engagement_edge/src/engagement_edge.py

```diff
@@ -23,14 +23,31 @@
 IS_LOCAL = bool(os.environ.get("IS_LOCAL", False))
 
 if IS_LOCAL:
-    JWT_SECRET = str(uuid.uuid4())
-    os.environ["BUCKET_PREFIX"] = "local-grapl"
+    import time
+    while True:
+        try:
+            secretsmanager = boto3.client(
+                "secretsmanager",
+                region_name='us-east-1',
+                aws_access_key_id="dummy_cred_aws_access_key_id",
+                aws_secret_access_key="dummy_cred_aws_secret_access_key",
+                endpoint_url='http://secretsmanager.us-east-1.amazonaws.com:4566'
+            )
+
+            JWT_SECRET = secretsmanager.get_secret_value(SecretId='JWT_SECRET_ID',)["SecretString"]
+            print(JWT_SECRET)
+            break
+        except Exception as e:
+            print(e)
```

Comment:
Let's initialize the logger earlier int he program and change this to a LOGGER.debug(e)